test/fixup-svg-string.js

```diff
@@ -142,3 +142,17 @@ test('fixupSvgString shouldn\'t correct non-image tags', t => {
     t.notEqual(dontFix.indexOf('img/png'), -1);
     t.end();
 });
+
+test('fixupSvgString SHOULD correct hrefs that are NOT data links', t => {
+    const fixed = fixupSvgString('<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:foo="http://www.w3.org/1999/xlink" xml:space="preserve"><image foo:href="https://placekitten.com/64/64"/></svg>');
+
+    t.equal(fixed.indexOf('https://placekitten.com/64/64'), -1);
```

Comment:
I think we should check for the whole string we expect -- to make sure the regex doesn't match too much